cl/settings/10-public.py

```diff
@@ -270,7 +270,10 @@
 # Email #
 #########
 if DEVELOPMENT:
-    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+    EMAIL_BACKEND = "cl.users.email_backend.EmailBackend"
+    BASE_BACKEND = "django_ses.SESBackend"
+    # Max email attachment in bytes, 350KB
+    MAX_ATTACHMENT_SIZE = 350000
```

Comment:
I think being an old man now, I can't count zeroes anymore and this format makes me happier (this was added in Python 3.6 or so?):

```suggestion
    MAX_ATTACHMENT_SIZE = 350_000
```

I'd also recommend putting the unit into the variable name: `MAX_ATTACHMENT_SIZE_BYTES` or similar.

Reply:
Changed!